### ISSUE TITLE
easyPlacement doesn't work on Realms

### ISSUE THREAD

**Issue by Draaen** [maruohon/litematica#664]
Im currently playing on the 1.19.3 version (Realms)
Here are the litematica and malilib versions: Litematica-fabric-1.19.3-0.13.1 (Malilib-fabric-1.19.3-0.14.0)
The issue is that when trying to place blocks, some work but some doesnt. I guess its a air placement issue for what i had read in the issue list, but anyone is talking about Realms when discussing this topic.

The block sounds as if placed but it just doesnt appear placed. The items seem to dissapear but later they come back in your inventory.
 
I have tried changing the hotbar (pickBlocableSlots) and tool change (pickBlockAvoidTools) mechanic but neither of those thing work.

https://user-images.githubusercontent.com/122884526/215742621-1af37621-3dc9-42fc-81a5-548c5344d030.mp4

**Comment by maruohon** [comment]
What is your `easyPlaceProtocolVersion` config set to? Only `Slabs Only` or `None` would work (as in place non-ghost blocks) on a Realm in 1.18.2+. `Auto` should fall back to `Slabs Only` on non-Carpet servers if I remember right. But if you have set it to `Version n 2` or `Version 3` then you would only place ghost blocks for any blocks that have some (supported) properties.

**Comment by Draaen** [comment]
I had easyPlaceProtocolVersion in Version 3. Now that I tried Auto, None and Slabs Only, the problem is that it kinda works but the blocks are not placed in the correct orientation.

https://user-images.githubusercontent.com/122884526/216011972-4238d415-09c3-4fc1-a026-108918bd8741.mp4

**Comment by maruohon** [comment]
Yes that is expected. Since there is no mod handling that "easy place protocol" on the server side, the mod can't request the correct orientation from the server.

There *may* be a player rotation spoofing feature at some point in the future, but for now on any server without a mod handling the Carpet mod "accurate placement protocol", easy place mode won't be able to rotate anything. (Other than blocks that only care about the block side you click on - and I think even that isn't implemented currently?)
